README.md

```diff
@@ -35,6 +35,27 @@ its features!
 Additionally, the [Litestar fullstack repository](https://github.com/litestar-org/litestar-fullstack)
 can give you a good impression how a fully fledged Litestar application may look.
 
+<details>
```

Comment:
Do we need that? GH already includes a toc. Maybe just a matter of personal taste, but if we do have a toc, I think it should be visible directly

Reply:
Maybe for mobile guys? GitHub App does not have TOC.